src/frontend/style/cart.css

```diff
@@ -0,0 +1,159 @@
+.cart {
+    max-width: 1200px;
+    margin: 2rem auto;
+}
+
+.cart-grid {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+}
```

Comment:
this could be named as `.grid-2-1` since its only function is to create a grid of 2:1 layout. Then you can use it with elements other than the `cart` as well

Reply:
yeah right, will keep this in mind from next time.